lib/linguist/languages.yml

```diff
@@ -4056,6 +4078,7 @@ Scheme:
   color: "#1e4aec"
   extensions:
   - ".scm"
+  - ".sch"
```

Comment:
Hrm, do you think it warrants one? These were the only [non-KiCad files](https://github.com/Alhadis/Linguist-Silo/blob/sch/breakdown.log#L2920-L2947) marked as KiCad Schematics, and [this](https://github.com/Alhadis/Linguist-Silo/blob/sch/breakdown.log#L743) was the only KiCad file identified as Scheme. All-in-all, I think the classifier's doing a good enough job. =)